tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -15756,15 +15793,17 @@ void TryAddShaderAttrFromTargetProfile(Sema &S, FunctionDecl *FD,
   }
 
   HLSLShaderAttr *currentShaderAttr = FD->getAttr<HLSLShaderAttr>();
-  // Don't add the attribute if it already exists as an attribute on the decl.
-  // In the special case that the target profile is compute and the
-  // entry decl already has a node shader attr, don't do anything
+  // Don't add the attribute if it already exists as an attribute on the decl,
+  // and emit an error. In the special case that the target profile is compute
+  // and the entry decl already has a node shader attr, don't do anything
```

Comment:
> ... In the special case that the target profile is compute
  // and the entry decl already has a node shader attr, don't do anything

There is no special case anymore.  We should get rid of this sentence entirely.